### PR TITLE
Expo v34

### PR DESCRIPTION
**Pull request by oxyii**
rel #85

### CODE REVIEW

**Comment by oxyii** [comment]
README edits needed

**Comment by oxyii** [comment]
Can anyone check? Everything seems to work for me. @Monte9 @iRoachie @xavier-villelegier

File: app.web-build.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    "slug": "react-native-elements-app",
+    "description": "A demo app for React Native Elements UI Library",
+    "privacy": "public",
+    "sdkVersion": "34.0.0",
+    "platforms": ["web"],
+    "version": "1.1.0",
+    "primaryColor": "#cccccc",
+    "web": {
+      "publicPath": "/react-native-elements-app/"
+    }
```
Comment (oxyii) [inline app.web-build.json:13]: Strange expo behavior. We must set correct `publicPath` for production build (eg for gh-pages) but also we must keep `publicPath` with default value (`/`) for dev-env. I can't find any documentation to solve this problem, so separated `app.json` used for builds only

Comment (iRoachie) [reply app.web-build.json:13]: Oh that is interesting

**Review by iRoachie — APPROVED** [review]
Looks good, tried it on all platforms.

I think sometime in the future we might need to rewrite this from scratch to improve performance on android.  But all good for now.

**Comment by oxyii** [comment]
@iRoachie We are ready to merge into `update`. Then please check README and we can merge all into `master` I think

File: README.md [hunk 2]
```diff
@@ -14,45 +14,44 @@ Also the RNE team runs and maintains this project, so if you need any help devel
...
-yarn native || npm run native
+# Using npm or yarn
+yarn || npm install
 ```
 
-4. Run the web app (uses [CRA](https://facebook.github.io/create-react-app/docs/getting-started))
+4. Run the cross-platform app (uses [Expo](https://expo.io/learn))
 
 ```
 yarn web || npm run web
```
Comment (iRoachie) [inline README.md]: Think this should be

```suggestion
yarn start || npm start
```

Comment (oxyii) [reply README.md]: @iRoachie  `yarn web` starts up Metro bundler for native platform and webpack for web but `yarn start` - just Metro, without webpack.

Comment (oxyii) [reply README.md]: We can tuning package.json's scripts and README inside `update` branch I think...

Comment (iRoachie) [reply README.md]: Yea but `yarn web` also opens the app in a browser. I think we should leave it as `yarn start`, the underlying `expo start` command shows you steps for running on web/ios/android.

yarn start
- press `i` (opens ios)
- press `a` (opens android)
- press `w` (opens web)

**Review by iRoachie — APPROVED** [review]

File: package.json [hunk 3]
```diff
@@ -1,58 +1,47 @@
...
-    "native": "expo start",
-    "build:web": "react-app-rewired build",
-    "deploy": "gh-pages -d build",
-    "build:android": "expo ba",
-    "build:ios": "expo bi",
-    "eject:web": "react-scripts eject",
-    "eject:native": "expo eject",
-    "prettify": "prettier --write 'src/**/*.js'",
-    "clean-install": "rm -rf node_modules && npm cache clean --force && watchman watch-del-all && yarn"
+    "rne-listitem-web": "gist:4a1da48acb977660e8665bae5262600b"
```
Comment (iRoachie) [inline package.json:36]: Didn't know you could do this, really cool!

Comment (oxyii) [reply package.json:36]: [Not first time](https://github.com/react-native-training/react-native-elements/issues/1871#issuecomment-485255900)

**Comment by iRoachie** [comment]
Cool i think we can merge

**Pull request merged**